src/core_editor/editor.rs

```diff
@@ -108,8 +108,8 @@ impl Editor {
         self.line_buffer.swap_graphemes();
     }
 
-    pub fn set_insertion_point(&mut self, line: usize, pos: usize) {
-        self.line_buffer.set_insertion_point(line, pos)
+    pub fn set_insertion_point(&mut self, pos: usize) {
+        self.line_buffer.set_insertion_point(pos)
```

Comment:
What is the interface here?

- do we have the idea of a present line and on that present line we are adding an insertion point?
- (or) are we saying that the outside world knows that internally we store this as a String and we get an index in that string?

Reply:
My assumption is that lines is a concept of the editor but not really of the outside world. We add a call that lets you set the insertion point to the closest it can find a row+col, also. This would be best effort, since the line may not have the column requested (or we may not have the line requested), but it could be one way of allowing someone to use it and build something more editor-like from it.